Comp/r-sas_ttest_Paired.qmd

```diff
@@ -0,0 +1,47 @@
+---
+title: "R vs SAS Paired T-Test"
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(procs)
+```
+
+# Paired t-test Comparison
+
+The following table shows the types of Paired t-test analysis, the capabilities of each language, and whether or not the results from each language match.
+
+| Analysis                      | Supported in R                            | Supported in SAS                       | Results Match    | Notes                                                 |
+|---------------|---------------|---------------|---------------|---------------|
+| Paired t-test, normal data    | [Yes](../R/ttest_Paired.html#normal)      | [Yes](../SAS/ttest_Paired.html#normal) | [Yes](#normal)   | In Base R, use `paired = TRUE` on `t.test()` function |
+| Paired t-test, lognormal data | [Maybe](../R/ttest_Paired.html#lognormal) | [Yes](../SAS/ttest_Paired.html#lognormal) | [NA](#lognormal) | May be supported by **envstats** package              |
+
+## Comparison Results
+
+### Normal Data {#normal}
+
+Here is a table of comparison values between `t.test()`, `proc_ttest()`, and SAS `PROC TTEST`:
+
+| Statistic          | t.test() | proc_ttest() | PROC TTEST | Match | Notes |
+|--------------------|----------|--------------|------------|-------|-------|
+| Degrees of Freedom | 11       | 11           | 11         | Yes   |       |
+| t value            | -1.0896  | -1.09        | -1.09      | Yes   |       |
```

Comment:
Can you please use the unrounded values rather than display values when looking at comparisons

```suggestion
| t value            | -1.089648	|-1.089648    | -1.089648	     | Yes   |      |
```